tests/unit/test_attribute_mapping.py

```diff
@@ -1,51 +1,71 @@
-from unittest import TestCase
+import pytest
+import responses
 
 from tamr_unify_client.project.attribute_mapping.resource import AttributeMapping
 
 
-class TestAttributeMapping(TestCase):
-    def test_resource(self):
-        test = AttributeMapping(self.mappings_json)
+@pytest.fixture
+def client():
+    from tamr_unify_client import Client
```

Comment:
Are there cyclic import issues with putting these `import`s at the top of the file? dynamic imports are usually only idiomatic to avoid cyclic imports

Reply:
No issue with cyclic imports; I was motivated purely by removing namespace clutter at the file level.